TravelAgencyService/Inerfaces/ITravelAgency.cs

```diff
@@ -0,0 +1,9 @@
+namespace TravelAgencyService;
+
+public class ITravelAgency
```

Comment:
Did you want to create interface here? Or class?

Reply:
If interface - it should be interface, instead I would make it abstract to avoid creation of such instances